### DISCUSSION TITLE
Crazyflie Bolt 1.1 for only one motor

### DISCUSSION THREAD

**Discussion by kyoeun-space** [bitcraze/discussions#703] [Q&A]
I want to make a drone with only one propeller (motor) using crazyflie bolt 1.1.
Is there a way that crazyflie supports this? If so, please let me know how to do it.

**Comment by tobbeanton** [comment]
You mean something like [this](https://www.bitcraze.io/tag/monocopter/)?

If so yes but it would require some work. Perhaps you can find what they did reading the papers or even a repository with the code.

**Comment by kyoeun-space** [comment]
yeah that's similar to what i want to make!
Thank you!!! 

-----Original Message-----
